game_event/models.py

```diff
@@ -51,3 +51,15 @@ def validate_game_event(court, ball_game, min_number_of_players, max_number_of_p
     @staticmethod
     def is_ball_game_playable_at_court(court, ball_game):
         return CourtBallGame.is_ball_game_playable(court, ball_game)
+
+    def create_with_admin(self, player, ball_responsible):
+        from game_event_player.models import GameEventPlayer
+        GameEventPlayer(game_event=self, player=player, ball_responsible=False).save()
+
+    def delete_game_event(self):
+        deleted = False
+        from game_event_player.models import GameEventPlayer
```

Comment:
Like Naor told you above' please move this import to the top of this file to avoid duplication